test/NuGet.Core.Tests/NuGet.Protocol.Tests/V2FeedParserTests.cs

```diff
@@ -660,7 +660,7 @@ public async Task V2FeedParser_NexusFindPackagesByIdNullDependencyRange()
             Assert.Equal(DateTimeOffset.Parse("2016-04-06T12:46:30.942Z"), latest.Published.Value);
             Assert.Equal(DateTimeOffset.Parse("2016-04-06T12:46:30.942Z"), latest.Created.Value);
             Assert.Equal(DateTimeOffset.Parse("2017-04-06T12:46:30.942Z"), latest.LastEdited.Value);
-            Assert.Equal("PackageB:null|EntityFramework:6.1.3|PackageC:3.7.0.15", latest.Dependencies);
+            Assert.Equal("PackageB:null|EntityFramework:6.1.3|PackageC:3.7.0.15|PackageD:3.14.15:null", latest.Dependencies);
             Assert.Equal(1, latest.DependencySets.Count());
```

Comment:
Without the fix, this line throws because the `latest.DependencySets` (incorrectly) has a count of 2, because `PackageD` is not grouped with all other dependencies.

Reply:
Hi @omajid, thanks for adding the test :)